cogs/gaminator.py

```diff
@@ -10,6 +10,23 @@
 TAB_LENGTH = 10
 MAX_LINES_PER_PAGE = 7
 
+
+def get_longest_role_name(role_dict):
+    longest_role = 0
+    for role, role_options in role_dict.items():
+        for role_option in role_options:
+            if len(role_option) > longest_role:
+                longest_role = len(role_option)
+    return longest_role
+
+
+def get_tab_length(default, longest_item, buffer=4):
+    return default if default > longest_item else longest_item + buffer
+
+
+DESTINY_ROLE_TAB_LENGTH = get_tab_length(TAB_LENGTH, get_longest_role_name(ROLES['game_modes']))
```

Comment:
Cool cool, maybe we should think about writing a padding support module?

Reply:
Sounds good, definitely something potentially more formal since that seems like a general formatting problem worth solving.